prime-router/src/main/kotlin/fhirengine/utils/FHIRBundleHelpers.kt

```diff
@@ -67,24 +67,17 @@ private fun lookupCondition(code: Coding, metadata: Metadata): Coding? {
  */
 fun Observation.getCodeSourcesMap(): Map<String, List<Coding>> {
     val toReturn = mutableMapOf<String, List<Coding>>()
```

Comment:
not crazy about the variable name `toReturn`. Maybe `codeSourcesMap` since thats what were getting?

Reply:
Agreed, will update the name